cylc/flow/network/schema.py

```diff
@@ -703,16 +703,23 @@ class Meta:
     pruned = Boolean()
 
 
+class RuntimeSetting(ObjectType):
+    """Key = val setting, e.g. for [runtime][X][environment]"""
```

Comment:
This will become user facing in the GUI so maybe make it a little more explicit.

```suggestion
    """Key = value setting for a `[runtime][<namespace>]` configuration."""
```